pkg/pdatatest/plogtest/logs.go

```diff
@@ -96,12 +96,7 @@ func CompareLogs(expected, actual plog.Logs, options ...CompareLogsOption) error
 // CompareResourceLogs compares each part of two given ResourceLogs and returns
 // an error if they don't match. The error describes what didn't match.
 func CompareResourceLogs(expected, actual plog.ResourceLogs) error {
-	var errs error
-
-	if !reflect.DeepEqual(expected.Resource().Attributes().AsRaw(), actual.Resource().Attributes().AsRaw()) {
-		errs = multierr.Append(errs, fmt.Errorf("attributes don't match expected: %v, actual: %v",
-			expected.Resource().Attributes().AsRaw(), actual.Resource().Attributes().AsRaw()))
-	}
+	errs := internal.CompareResource(expected.Resource(), actual.Resource())
 
```

Comment:
Does `SchemaUrl` in `ResourceSpans`, `ResourceLogs`, `ResourceMetrics` should be compared in `CompareResourcexxx` functions? @dmitryax

Reply:
Yeah, I also just found this out.

Reply:
Updated. PTAL